planetarium/NineChronicles
# Fix crafting balance v2

### Description

1. [Fix crafted equipment's skill bug](https://app.asana.com/0/1133453747809944/1200807881396804/f)
2. Fix weapon's skill variations

### How to test

1. Check tables.
2. Check item's options in the craft menu.

### Related Links

### Required Reviewers

@Namyujeong 

### Screenshot
![image](https://user-images.githubusercontent.com/60082897/130022039-fc89b13e-b154-48fb-adb4-3716651b8a43.png)
